docs/cep/query-guide/functions/time/utcTimestamp.md

```diff
@@ -26,7 +26,7 @@ This query returns the system current time in UTC timezone with `yyyy-MM-dd HH:m
 
 ```sql
 CREATE STREAM InputStream ();
```

Comment:
We can't use CREATE STREAM with empty parameters.. we need to specify something here inside the bracket.. For example, CREATE STREAM InputStream (message string); 
then pass the same message to Output stream:
CREATE SINK STREAM OutputStream (message string, utcTimestamp string);
@info(name = 'utcTimestampQuery')
INSERT INTO OutputStream
SELECT message, time:utcTimestamp() AS utcTimestamp
FROM InputStream;

Then finally add that message part in the explanation as well?